tests/Unit/Core/Search/Builder/TypedBuilder/ProductCombinationBuilderTest.php

```diff
@@ -54,20 +54,22 @@ public function testSupports(string $filtersClass, bool $expectedSupport)
     public function testBuildFilters()
     {
         $productId = 42;
+        $shopId = 2;
         $builder = new ProductCombinationFiltersBuilder();
```

Comment:
The test class should probably be renamed `ProductCombinationFiltersBuilderTest` (the `Filters` part is missing)